.pulumi-java-gen.version

```diff
@@ -1 +1 @@
-0.14.0
\ No newline at end of file
+0.16.1
```

Comment:
Did the provider upgrade fail with 0.14.0?  The CI job avoids upgrading pulumi-java so that we can be confident we are only reviewing changes from the provider and not from elsewhere.  Ideally we should replicate that behavior when manually upgrading the upstream provider.

Reply:
I executed the command `upgrade-provider equinix/pulumi-equinix --kind=provider --target-bridge-version=latest --pr-assign=ocobles --pr-title-prefix=feat:` without the `--java-version` arg and it seems that if you don't specify the java version it updates to the latest version available. However, I don't see any changes in the SDK generation and it fixes some bugs internally https://github.com/pulumi/pulumi-java/compare/v0.14.0...v0.16.1